app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ServerSuggestion.kt

```diff
@@ -0,0 +1,18 @@
+package org.jellyfin.mobile.ui.screens.connect
+
+data class ServerSuggestion(
+    val type: Type,
+    val name: String,
+    val address: String,
+    /**
+     * A timestamp for this suggestion, used for sorting.
+     * For discovered servers, this should be the discovery time,
+     * for saved servers, this should be the last used time.
+     */
+    val timestamp: Long,
```

Comment:
Didn't catch this earlier, I would use an Instant here so it's more typesafe and less prone to issues with formatting/timezones/etc. Although that would likely need changes to the db scheme and for its current usage a long works fine.

Reply:
Right, an instant would definitely be better, but at the moment we only use it for sorting and as long as we don't change the database changing the rest doesn't make a lot of sense.